packages/common/browser/gated-download-form-dot-com.vue

```diff
@@ -34,6 +38,7 @@ export default {
     window.addEventListener('message', (e) => {
       if (typeof e.data === 'string' && e.data.indexOf('type:whitepaper-registration') !== -1) {
         this.canDownload = true;
+        window.open(this.target);
```

Comment:
Test this in IE11 (if possible)

Reply:
Confirmed with Chris Green, working in IE11